### ISSUE TITLE
Enhancement for fan status

### ISSUE THREAD

**Issue by jskarp5953** [hugheaves/mios_wifi-thermostat#1]
Was wondering if this plugin could be updated with fan status. I would like to create a fan status trigger to turn on and off devices when the fan comes on or shutdown.

**Comment by hugheaves** [comment]
Yes, absolutely. This is a simple modification, and I just need to find a few seconds to add it to the code.

**Comment by grndamgt4** [comment]
+1. I would like to turn on a UV light in the duct work when the fan is on. Thanks!
